pkg/kubelet/kubeletconfig/util/equal/equal.go

```diff
@@ -1,24 +0,0 @@
-/*
```

Comment:
I would be happy to hear suggestions from the community on whether the current file (package) should be deleted or retained.